# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/base/content/browser.xhtml
# Commit: 72e19189bb99
# Full Hash: 72e19189bb99e159b2974cbcda69647211962df3
# Author: Brendan Dahl <bdahl@mozilla.com>
# Date: 2019-06-14 03:39:39
# Regressor Bug: 1552049
# File Overlap Count: 1
# Description:
#   Bug 1552049 - Enable XUL persistence for all system privilege docs. r=smaug
#   
#   Remove the mozpersist attribute and XULDocument requirement to use XUL
#   persistence and allow any system privilege document to use it by default.
#   
# ==============================================================================

diff -r bc46a287b898 -r 72e19189bb99 browser/base/content/browser.xhtml
--- a/browser/base/content/browser.xhtml	Thu Jun 13 17:37:15 2019 +0000
+++ b/browser/base/content/browser.xhtml	Thu Jun 13 18:15:54 2019 +0000
@@ -70,7 +70,6 @@
         sizemode="normal"
         retargetdocumentfocus="urlbar"
         persist="screenX screenY width height sizemode"
-        mozpersist=""
         >
 
 # All JS files which are needed by browser.xhtml and other top level windows to